src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -39,6 +40,23 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("Exceptions", LoaderExceptions, typeof(Exception[]));
         }
 
+        /// <summary>
+        /// Creates and returns a string representation of the current <see cref="AggregateException"/>.
```

Comment:
This summary is still referencing AggregateException. 

I do not thing the comment is needed here at all because of it has zero value, and no other methods in the file have it.